dpctl-capi/helper/include/dpctl_string_utils.hpp

```diff
@@ -50,6 +50,10 @@ cstring_from_string(const std::string &str)
 #else
         std::strncpy(cstr, str.c_str(), cstr_len);
 #endif
+        // This is redundant because str.c_str() is guaranteed to be
+        // null-terminated and copy function is asked to copy that
+        // null-character.
```

Comment:
Will be good to add that we still need the line for the specific reason of satisfying Checkmarx. Otherwise it may confuse folks as to why we ever added the line knowingly.

Reply:
Comment revised.